README.md

```diff
@@ -0,0 +1,46 @@
+# @sveltejs/branding
+
+Logos and Logotypes for Svelte in different formats.
+
+`svelte-logo.pdf` or `sapper-logo.pdf` are editable Adobe-Illustrator-CC-files which can be useful for occasions where none of the provided solutions fit.
+
+`colors.ase`  is the exported svelte color-palette [Adobe CC]
+
+**Don't** alter paths or proportions in any way.
+
+
+
+### Icons
+
+* Icons are available at sizes 16, 24, 32, 48, 64 or 128px
+* Either 1x or 4x resolution
+* .png or .svg
+
+<img style="width:32px; height:32px" src="https://github.com/sveltejs/branding/icons/svelte.svg" alt="svelte icon 32x32" />
+
+
+
+### Logos | Press
+
+- prefered format is `svelte-horizontal`
+- use `svelte-vertical` if your Layout isn't suited a for the use of `svelte-horizontal` 
+- for print you'll have to separate the colors from RGB to 4c yourself, according to your print-process
+
+<img style="width:300px" src="https://github.com/sveltejs/branding/press/svelte-horizontal.svg" alt="svelte-horizontal logo" />
+
+
+
+### Colors
+
+<table>
+	<td style="width:120px; height:120px; padding: .8em 1.2em; color:white; background:#ff3e00;">#ff3e00</td>
+<td style="width:120px; height:120px; padding: .8em 1.2em; color:white; background:#009498;">#009498</td>
+<td style="width:120px; height:120px; padding: .8em 1.2em; color:white; background:#4a4a55;">#4a4a55</td>
+</table>
+
+
+
+## License
+
+[]()
```

Comment:
missing Licence

Reply:
Yeah. I'm unsure about this and asked in the comment.

Reply:
Oops, right, sorry.

I had a look at creative commons (https://creativecommons.org/choose/):
- if you don't want others to derive anything from the logos there's [Attribution-NonCommercial-NoDerivatives 4.0 International](https://creativecommons.org/licenses/by-nc-nd/4.0/)
- if you're ok with others making new things with the logos there's [Attribution-NonCommercial-ShareAlike 4.0 International](https://creativecommons.org/licenses/by-nc-sa/4.0/)